### PR TITLE
Disconnectcol

### PR DESCRIPTION
**Pull request by KarlTD**
* Fixed Ride shutdown when APL task disappears.
* Changed session background color when above occur. The suggested colors are various hues and tints of red depending on theme to preserve contrast.

### CODE REVIEW

**Comment by e9gille** [comment]
@abrudz I'd like your opinion on the color changes seeing as you provided some of those themes, please :)

**Comment by abrudz** [comment]
Maybe make
Dracula: #7f2a2a
New Moon: #7a3b3d
Nord: #603136
These are half-brightness of the respective themes' reddest colour.

File: src/prf_col.js [hunk 1]
```diff
@@ -95,17 +95,17 @@
...
       theme: 'light',
       styles: 'com=I:1 diam=B:1 err=fg:0,bg:1,bgo:.5,B:1,I:1 glb=I:1 kw=B:1 '
       + 'lnum=bg:f,bgo:0 mod=bg:7,bgo:.25 mtch=bg:c,bgo:.5 norm=bg:f,bgo:1 ns=fg:8 num=fg:8 quad=fg:8 srch=bg:c,bgo:.5 '
-      + 'str=fg:8 tc=bg:e,bgo:1 tcpe=bg:dadada,bgo:1 zld=fg:8 vtt=bc:aaa dc=bg:e2d3bb,bgo:1',
+      + 'str=fg:8 tc=bg:e,bgo:1 tcpe=bg:dadada,bgo:1 zld=fg:8 vtt=bc:aaa dc=bg:#993333,bgo:1',
     }, {
       name: 'Kazimir Malevich',
       theme: 'light',
-      styles: 'norm=bg:f,bgo:1',
+      styles: 'norm=bg:f,bgo:1,bgo:1 zld=fg:8 vtt=bc:aaa dc=bg:#993333',
```
Comment (e9gille) [inline src/prf_col.js:108]: This one looks like you duplicated `bgo` and perhaps added the other entries unintentionally?

**Review by e9gille — CHANGES_REQUESTED** [review]

File: src/prf_col.js (hunk 1 above)
Comment (KarlTD) [reply src/prf_col.js]: Thank you for spotting this.

**Review by e9gille — APPROVED** [review]

**Pull request merged**